pgjdbc/src/test/java/org/postgresql/test/TestUtil.java

```diff
@@ -211,14 +212,22 @@ public static void initDriver() throws Exception {
       p.putAll(System.getProperties());
       System.getProperties().putAll(p);
 
-      if (getLogLevel() > 0) {
-        // Ant's junit task likes to buffer stdout/stderr and tends to run out of memory.
-        // So we put debugging output to a file instead.
-        java.io.Writer output = new java.io.FileWriter("postgresql-jdbc-tests.debug.txt", true);
-        java.sql.DriverManager.setLogWriter(new java.io.PrintWriter(output, true));
+      if (org.postgresql.Driver.isRegistered()) {
+        LogManager manager = LogManager.getLogManager();
+        java.util.logging.Logger logger = manager.getLogger("org.postgresql");
+        switch (getLogLevel()) {
```

Comment:
I wonder if this is part of TestUtil required at all.
Wouldn't it be nice if exactly the same log-enablement approach was used for both unit testing and production purposes?